force_wfmanager/wfmanager_setup_task.py

```diff
@@ -343,7 +295,8 @@ def sync_review_task(self):
             for task in self.window.tasks:
                 if task.name == "Review":
                     self.review_task = task
-                    self.review_task.run_enabled = self.run_enabled
+                elif task.name == "Workflow Setup":
```

Comment:
I realize that this is duplicating the approach of existing code, but in this case I would recommend using the `task.id` for both tests rather than the `task.name`.

Don't need to fix in this PR if the problem occurs in many places, but if you don't fix it, open an issue.